cypress/e2e/EngageSphere.cy.js

```diff
@@ -0,0 +1,41 @@
+describe('Test suite for Engage Sphere Website', () => {
+
+  beforeEach(() => {
+    cy.visit(Cypress.config('baseUrl'))
+  })
+
+  it('Renders the header with an h1 and theme toggle', () => {
+    cy.get('h1').should('have.text', 'EngageSphere');
```

Comment:
Um texto pode estar no DOM sem estar visível.

Qua tal o seguinte?

```suggestion
    cy.contains('h1', 'EngageSphere').should('be.visible');
```